.github/CODEOWNERS

```diff
@@ -12,96 +12,97 @@
+/.github/actions/                                                                     @island-is/devops
+/.github/workflows/                                                                   @island-is/devops
+/apps/reference-backend/                                                              @island-is/devops
+/libs/infra*/                                                                         @island-is/devops
+/libs/logging/                                                                        @island-is/devops
+/scripts/                                                                             @island-is/devops
+/scripts/postinstall.js                                                               @island-is/devops @jeremybarbet
+/scripts/schemas.js                                                                   @island-is/devops @jeremybarbet
 
-/libs/api/domains/api-catalogue/                                    @island-is/advania
-/libs/api/domains/document-provider/                                @island-is/advania
-/libs/api-catalogue/                                                @island-is/advania
-/libs/application/templates/document-provider-onboarding/           @island-is/advania
-/libs/service-portal/document-provider/                             @island-is/advania
-/apps/services/xroad-collector/                                     @island-is/advania
-/apps/services/documents                                            @island-is/advania
+/libs/api/domains/api-catalogue/                                                      @island-is/advania
+/libs/api/domains/document-provider/                                                  @island-is/advania
+/libs/api-catalogue/                                                                  @island-is/advania
+/libs/application/templates/document-provider-onboarding/                             @island-is/advania
+/libs/service-portal/document-provider/                                               @island-is/advania
+/apps/services/xroad-collector/                                                       @island-is/advania
+/apps/services/documents                                                              @island-is/advania
 
-/apps/skilavottord/                                                 @island-is/deloitte
-/libs/application/templates/health-insurance                        @island-is/deloitte
-/libs/skilavottord/                                                 @island-is/deloitte
-/libs/api/domains/health-insurance/                                 @island-is/deloitte
-/libs/health-insurance/                                             @island-is/deloitte
+/apps/skilavottord/                                                                   @island-is/deloitte
+/libs/application/templates/health-insurance                                          @island-is/deloitte
+/libs/application/template-api-modules/src/lib/modules/templates/health-insurance     @island-is/deloitte
```

Comment:
I think you need backslash at the end, no?

Reply:
Done fixed